client/src/components/Message/Message.js

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import "./Message.css";
+import ProfilePicture from "../ProfilePicture/ProfilePicture";
+import defaultPic from "assets/images/profile-picture-1.png";
+import defaultPic2 from "assets/images/profile-picture-2.png";
+
+import ExtraUsers from "../ExtraUsers/ExtraUsers";
+
+// Function based React Component
+const Message = props => {
```

Comment:
This component is pretty long and the way it's currently written is a little complex. Can you simplify this a bit?

Reply:
Changed the .map for names into if statements and refactored component (also addresses issue #245)